hoodie-client/src/main/java/com/uber/hoodie/index/hbase/HBaseIndex.java

```diff
@@ -233,42 +307,66 @@ private Connection getHBaseConnection() {
       }
       return writeStatusList.iterator();
     }
+
+    /**
+     * Helper method to facilitate performing puts and deletes in Hbase
+     * @param hTable
+     * @param puts
+     * @param deletes
+     * @throws IOException
+     */
+    private void doPutsAndDeletes(HTable hTable, List<Put> puts, List<Delete> deletes) throws IOException {
+      if(puts.size() > 0) {
+        hTable.put(puts);
+      }
+      if(deletes.size() > 0) {
+        hTable.delete(deletes);
+      }
+      hTable.flushCommits();
+      puts.clear();
+      deletes.clear();
+    }
   }
 
   @Override
   public JavaRDD<WriteStatus> updateLocation(JavaRDD<WriteStatus> writeStatusRDD,
-      HoodieTable<T> hoodieTable) {
+                                             HoodieTable<T> hoodieTable) {
     return writeStatusRDD.mapPartitionsWithIndex(new UpdateLocationTask(), true);
   }
 
   @Override
   public boolean rollbackCommit(String commitTime) {
-    // Can't really rollback here. HBase only can let you go from recordKey to fileID,
-    // not the other way around
+    // Rollback in HbaseIndex is managed via method {@link #checkIfValidCommit(String)}
     return true;
   }
 
   /**
    * Only looks up by recordKey
+   * @return
    */
-  @Override
```

Comment:
why remove `@Override` ?

Reply:
Happened during merge with internal repo vs open source, fixed it.